src/subscription_manager/ga_loader.py

```diff
@@ -46,46 +43,21 @@
 
 
 class GaImporter(object):
```

Comment:
Why don't you remove this completely?

Reply:
I didn't do it because GLib is used for dbus communication, and I didn't want to make unneccesary changes in the dbus `server.py` file. I can confirm that importing `gi.repository` objects instead of calling the import method does work as expected, it will be included in the next update of this PR.

The import method was also called in `rhsmcertd_worker.py` and `subscription_manager.py` scripts, but it does not seem it was used by anything.

Reply:
Oh, I can see that it is used in e.g. `rhsmcertd_worker.py`, but is this file really necessary to have?

Reply:
It is not, whole ga_loader logic is removed now.